hendrix/deploy/base.py

```diff
@@ -35,17 +35,21 @@ def __init__(self, action='start', options={}, reactor=reactor):
         # because running the management command overrides self.options['wsgi']
         if self.options['wsgi']:
             if hasattr(self.options['wsgi'], '__call__'):
-                # If it has a __call__, we assume that it is the application object itself.
+                # If it has a __call__, we assume that it is the application
+                # object itself.
                 self.application = self.options['wsgi']
                 try:
-                    self.options['wsgi_app_name'] = "%s.%s" % (self.application.__module__, self.application.__name__)
+                    self.options['wsgi'] = "%s.%s" % (
```

Comment:
Can you help me understand this change?  Are we still allowing them to either pass an application object or a dot path?